modules/extdata/uri/pom.xml

```diff
@@ -64,7 +64,27 @@
         <module>modules/uri-dependency</module>
     </modules>
 
+    <properties>
```

Comment:
What is the reason for extdata-uri changes? Is it a simple refactoring?

Reply:
from http://apache-ignite-developers.2346864.n4.nabble.com/Building-with-maven-3-8-1-tp52689.html

Building ignite-exdata-uri with
latest jar plugin produces errors like:

     ignite-extdata-uri: You have to use a classifier to attach supplemental artifacts to the project instead of replacing them

Seems that jar plugin doesn't like when build produces multiple jars
even if they finalName's are different. Reworking build configuration
with classifiers fixed the problem.

So. Yep i reworked a little config since we will have to use classifiers